src/cdk/stepper/stepper.ts

```diff
@@ -63,6 +64,37 @@ export class StepperSelectionEvent {
   previouslySelectedStep: CdkStep;
 }
 
+/** The state of each step. */
+export type StepState = 'number' | 'edit' | 'done' | 'error' | string;
```

Comment:
isn't this just the same as `string`?

Reply:
Sure, but like this it is clear to the developer that the default states that we are using are `'number' | 'edit' | 'done' | 'error'` and that the developer also has the option to define their own `state`.

Reply:
We can reference the enum here instead of repeating all the literals `STEP_STATE | string`